tripFrontend/src/features/product/components/ProductForm/ProductForm.tsx

```
@@ -1,6 +1,6 @@
 'use client';
 
-import { X, Save, Plus, Map } from 'lucide-react';
+import { X, Save, Plus, Map, Trash2 } from 'lucide-react';
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)